lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -278,13 +280,42 @@ defmodule ExUnit.Runner do
           end
       end
 
+    to_run = sort_first_last(config, to_run)
```

Comment:
I don't think these changes are necessary? We only sort files, not actual tests?

Reply:
If we only want to support sorting files, then yes. But as I am using the existing ExUnit filters, I saw the opportunity to also support `--sort-first=mytag` to sort tests themselves.

Reply:
I see. Let's start with either tags or files first. I think files will reduce complexity, so that gets my vote. :)

Reply:
Hmm so it would only remove the changes in ExUnit.Runner, but then I'd probably need to validate that the parameters given to `--sort-first/last` are only using `file:` filters, or would you recommend not using the ExUnit filter syntax and directly pass files:

Instead of:

```
mix test --sort-first file:path/to/test.exs --sort-first file:path/to/other/test.exs
```

just

```
mix test --sort-first path/to/test.exs --sort-first path/to/other/test.exs
```

?

Maybe test complexity would be reduced, I'll need to look at those.

Reply:
Exactly, just:

```
mix test --sort-first path/to/test.exs --sort-first path/to/other/test.exs
```